features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/resource-access-control-v2.xml.j2

```diff
@@ -832,12 +832,12 @@
         </Resource>
 
         <!-- [Organization] Admin Application Management API -->
-        <Resource context="(.*)/api/server/v1/applications/(.*)/owner(.*)" secured="true" http-method="PUT">
-            <Scopes>internal_organization_admin</Scopes>
+        <Resource context="(.*)/o/api/server/v1/applications/(.*)/owner(.*)" secured="true" http-method="PUT">
+            <Scopes>internal_org_organization_admin</Scopes>
         </Resource>
         <!-- Should this go to applications mgt API -->
-        <Resource context="(.*)/api/server/v1/organizations/(.*)/share" secured="true" http-method="POST">
-            <Scopes>internal_application_mgt_update</Scopes>
+        <Resource context="(.*)/o/api/server/v1/organizations/(.*)/share" secured="true" http-method="POST">
```

Comment:
Application sharing is not organization perspective API. The previous way looks correct.

Reply:
fixed